### ISSUE TITLE
Discrepancy dev dependencies with NativeScript Angular

### ISSUE THREAD

**Issue by jessorlisa** [NativeScript/nativescript-dev-webpack#1114]
**Is your feature request related to a problem? Please describe.**
 
I am working on a NativeScript Angular project and I am confused about the dev dependencies. When I update the dev dependencies a fixed dependency of `@angular/compiler-cli@8.2.0` is added
Output:
```
$ ./node_modules/.bin/update-ns-webpack --deps
Updating dev dependencies...
Adding dev dependency: @angular/compiler-cli@8.2.0
```
Updating the dev dependencies for @nativescript/angular results in a less strict requirement:
```
$ ./node_modules/.bin/update-app-ng-deps
[...]
Updated dependency @angular/compiler-cli to version: ~8.2.0.
```
Everything seems to be working fine with the latter.

**Describe the solution you'd like**
`$ ./node_modules/.bin/update-ns-webpack --deps` adds the same dependencies as `$ ./node_modules/.bin/update-app-ng-deps` (it does not matter which one).

**Describe alternatives you've considered**
Ignoring the discrepancy ;)

**Additional context**
```
$ tns info
✔ Getting NativeScript components versions information...
✔ Component nativescript has 6.3.1 version and is up to date.
✔ Component @nativescript/core has 6.3.2 version and is up to date.
✔ Component tns-android has 6.3.1 version and is up to date.
✔ Component tns-ios has 6.3.0 version and is up to date.
```

```
$ npm ls nativescript-dev-webpack
└── nativescript-dev-webpack@1.4.1 
```

**Comment by rosen-vladimirov** [comment]
Hey @jessorlisa ,
Thanks for pointing this difference. The code for updating dependencies via ns-update-webpack script is available [here](https://github.com/NativeScript/nativescript-dev-webpack/blob/81ac4aa30136a9145803d97a0adb8320cda4d18e/dependencyManager.js#L70-L84). Maybe you can send us a PR to resolve this issue?

**Comment by jessorlisa** [comment]
Sure, here you go.

**Comment by dtopuzov** [comment]
Fix is already available in `nativescript-dev-webpack@next`.
Will be officially released with  `nativescript-dev-webpack@1.5.0`
